test/render.test.js

```diff
@@ -289,5 +289,49 @@ describe('PostHTML Render', function () {
         expect(render(fixture, options)).to.eql(expected)
       })
     })
+
+    describe('quoteAllAttributes', function () {
+      it('True', function () {
+        var options = { quoteAllAttributes: true }
+
+        var fixture = { tag: 'a', attrs: { href: '/about/me/' } }
+        var expected = '<a href="/about/me/"></a>'
+
+        expect(render(fixture, options)).to.eql(expected)
+      })
+
+      it('False', function () {
+        var options = { quoteAllAttributes: false }
+
+        var fixture = { tag: 'a', attrs: { href: '/about/me/' } }
+        var expected = '<a href=/about/me/></a>'
+
+        expect(render(fixture, options)).to.eql(expected)
+      })
+
+      it('Required', function () {
+        var options = { quoteAllAttributes: false }
+
+        var fixture = { tag: 'a', attrs: { href: '/about-\t-characters' } }
+        var expected = '<a href="/about-\t-characters"></a>'
```

Comment:
can you add tests for 

```js
     {
       "tag": "p",
       "attrs": {
          "id": "asd adsasd"
        },
     },
```

Reply:
Ok this will work with the fixture I mentioned.  👍 

All good 👍

Reply:
Done.